Source/Blazorise/Enums/FloatingPositionStrategy.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Blazorise.Enums;
+
+/// <summary>
+/// Defines the position strategy for the floating component.
+/// <para>See : https://floating-ui.com/docs/computeposition#strategy</para>
+/// </summary>
+public enum FloatingPositionStrategy
```

Comment:
I would name this DropdownMenuPositionStrategy, or DropdownPositionStrategy as it is used only on the Dropdown.

Reply:
This way it's future proof though.
I don't quite remember, but don't we have something else using it? tooltips?

Reply:
I guess you're right. I'll change it. In the future if we need to use elsewhere, we can just do a copy paste for naming sake